### PR TITLE
bump to version 1.0, add option for ?frame query parameter with test

### PR DESCRIPTION
**Pull request by nmonterroso**
@drsnyder 
https://wikia-inc.atlassian.net/browse/PLATFORM-984

Java support for `frame` query parameter in https://github.com/Wikia/vignette/pull/79

### CODE REVIEW

**Comment by drsnyder** [comment]
lgtm

File: lib/vignette/src/main/java/com/wikia/vignette/UrlGenerator.java [hunk 1]
```diff
@@ -224,6 +224,24 @@ public Builder fill(String fill) {
       return this;
     }
 
+    /**
+     * for animated gifs, capture a specific frame of animation
+     * @param frame
+     * @return UrlGenerator.Builder
+     */
+    public Builder frame(int frame) {
```
Comment (drsnyder) [inline lib/vignette/src/main/java/com/wikia/vignette/UrlGenerator.java]: Minor: any reason why you used `int` and not `Integer`?

Comment (nmonterroso) [inline lib/vignette/src/main/java/com/wikia/vignette/UrlGenerator.java]: Why would you use `Integer` and not `int`? :P

Comment (drsnyder) [inline lib/vignette/src/main/java/com/wikia/vignette/UrlGenerator.java]: The convention in the file seems to be the boxed types (`Integer` see `width` below). On the line below the `toString` method is also being used so I imagine the static `Integer.toString(frame)` is probably creating a boxed type underneath anyway. That line could also be simplified to `frame.toString()` if the boxed version was used.

/end nitpick

**Pull request merged**